src/sass/map.scss

```diff
@@ -0,0 +1,112 @@
+// Mixins
+@mixin mapbox-ctrl-badge($content) {
+    &::before {
+        content: $content;
+        background: black;
+        color: white;
+        margin-right: 4px;
+        position: absolute;
+        right: 37px;
+        font-size: 14px;
+        white-space: nowrap;
+        padding: 5px 9px;
+        border-radius: $mapbox-ctrl-radius;
+    }
+}
+
+@mixin mapbox-ctrl-icon($name, $content) {
+    $white-icon: "#{$name}-white";
+    .ons-ctrl-#{$name} {
+        .mapboxgl-ctrl-icon {
+            background-image: url("data:image/svg+xml, #{getSVG($name)}");
+            &:hover {
+                background-image: url("data:image/svg+xml, #{getSVG($white-icon)}");
+                @include mapbox-ctrl-badge($content);
+            }
+            @content;
+        }
+    }
+}
+
+// Mapbox styles
+#map {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    max-height: $map-width;
+    @media screen and (min-width: $md-breakpoint) {
```

Comment:
I have put a note in the README.md to point out that this can be overridden. The other question do we care about setting these map dimensions here in the library OR if we use the webpack build with sass/ css loaders then we can set these style rules in the web apps directly?